common.gypi

```diff
@@ -10,6 +10,8 @@
     'component%': 'static_library',   # NB. these names match with what V8 expects
     'msvs_multi_core_compile': '0',   # we do enable multicore compiles, but not using the V8 way
     'python%': 'python',
+    'node_engine%': 'v8',
```

Comment:
Nit: is `node_engine` established elsewhere? Maybe `js_engine`? This line and everywhere else for consistency